site2/docs/io-redis-sink.md

```diff
@@ -55,15 +55,81 @@ Before using the Redis sink connector, you need to create a configuration file t
 * YAML
 
     ```yaml
-    {
+    configs:
         redisHosts: "localhost:6379"
-        redisPassword: "fake@123"
-        redisDatabase: 1
+        redisPassword: "mypassword"
+        redisDatabase: 0
         clientMode: "Standalone"
         operationTimeout: 2000
-        batchSize: 100
+        batchSize: 1
         batchTimeMs: 1000
         connectTimeout: 3000
-    }
+    ```
+  
+### Usage
+
+This example shows how to write records to a Redis database using the Pulsar Redis connector.
+
+1. Start a Redis server.
+
+    ```bash
+    $ docker pull redis:5.0.5
+    $ docker run -d -p 6379:6379 --name my-redis redis:5.0.5 --requirepass "mypassword"
```

Comment:
Remove $ so that users can copy the commands more easily. Pls check all.

Reply:
It is better to use `$` since it is a executable command line and coders can get it as other usage in docs do.

Reply:
what the meaning of "coders can get it as other usage in docs do."?

Reply:
I mean that programmers usually use  `$` to indicate executable commands and other `md` examples use this style.

Reply:
In Pulsar docs, some contents are not user-friendly. We are improving UX and making changes to docs.

Reply:
Good point, however, a command line started with `$`  is more popular and commonly used, since some users may confuse executable commands with comment lines.

Reply:
"comment lines" means "annotation"? if so, i think users may not be confused since we use "//" for annotation.